### DISCUSSION TITLE
I have multiple files written in asciidoc and i am unable to use the to fill my nuxt  blog

### DISCUSSION THREAD

**Discussion by amineHY** [nuxt/nuxt#19176] [Questions] [answered]
As explain in the title, I like writing using asciidoc instead of markdown, and to my surpise, there is no support for ascii format in nuxt framework, am I missing something or is there a workarround ?  Thanks

**Comment by Baroshem** [comment, marked as answer]
Hey, 

Nuxt framework does not work with Markdown by default but instead with Vue.js components. For Markdown support, users usually use the NuxtContent -> https://content.nuxtjs.org/

I think you should ask there. Guys from content should help you out :)
